docs/src/catalyst_applications/bifurcation_diagrams.md

```diff
@@ -1,115 +1,118 @@
+- The bifurcation parameter (the parameter which is varied in the bifurcation diagram).
+- A full model parameter set. This includes the values of all non-bifurcation parameters, but also a value for the bifurcation parameter (which correspond to the point in parameter space from which the computation of the bifurcation diagram starts).
+- An initial guess of the steady state values of the system at the provided parameter set. Using this point as a start, BifurcationKit uses Newton's method to find an initial steady state from which to compute the bifurcation diagram. Hence, this guess does not need to be very exact (but may be important if the system exhibits multistability for the initial parameter set).
+- The species which concentration we wish to plot on the y-axis of the bifurcation diagram (alternatively, a custom value can be provided by using the [`record_from_solution` argument](https://bifurcationkit.github.io/BifurcationKitDocs.jl/stable/periodicOrbit/#.-record*from*solution)).
+
+We combines all this information to form a `BifurcationProblem`:
 ```@example ex1
-p = Dict(:S => 1., :D => 9., :τ => 1000., :v0 => 0.01,
-         :v => 2., :K => 20., :n => 3, :d => 0.05)
-bif_par = :S           # bifurcation parameter
-p_span = (0.1, 20.)    # interval to vary S over
-plot_var = :X          # we will plot X vs S
-nothing   # hide
+using BifurcationKit
+bif_par = :k1
+u_guess = [:X => 5.0, :Y => 2.0]
+p_start = [:k1 => 4.0, :k2 => 1.0, :k3 => 1.0, :k4 => 1.5, :k5 => 1.25]
+plot_var = :X
+bprob = BifurcationProblem(wilhelm_2009_model, u_guess, p_start, bif_par; plot_var=plot_var)
+nothing # hide
 ```
-When creating a bifurcation diagram, we typically start at some point in
-parameter-space. We will simply select the beginning of the interval over which
-we wish to compute the bifurcation diagram, `p_span[1]`. We thus create a
-modified parameter set where `S = 0.1`. For this parameter set, we also guess
-the steady state of the system. While a good estimate could be provided through
-an ODE simulation, BifurcationKit does not require the guess to be very
-accurate.
+
+BifurcationKit computes bifurcation diagrams using the `bifurcationdiagram` function. From an initial point in the diagram, it tracks the solution (using a continuation algorithm) until the entire diagram is computed (BifurcationKit's continuation can be used for other purposes, however, this tutorial focuses on bifurcation diagram computation). The continuation settings are provided in a `ContinuationPar` structure. In this example, we will only specify three settings, `p_min` and `p_max` (which sets the minimum and maximum values over which the bifurcation parameter is varied) and `max_steps` (the maximum number of continuation steps to take as the bifurcation diagram is tracked). We wish to compute a bifurcation diagram over the interval *(2.0,20.0)*, and will use the following settings:
```

Comment:
```suggestion
BifurcationKit computes bifurcation diagrams using the `bifurcationdiagram` function. From an initial point in the diagram, it tracks the solution (using a continuation algorithm) until the entire diagram is computed (BifurcationKit's continuation can be used for other purposes, however, this tutorial focuses on bifurcation diagram computation). The continuation settings are provided in a `ContinuationPar` structure. In this example, we will only specify three settings, `p_min` and `p_max` (which sets the minimum and maximum values over which the bifurcation parameter is varied) and `max_steps` (the maximum number of continuation steps to take as the bifurcation diagram is tracked). We wish to compute a bifurcation diagram over the interval `(2.0,20.0)`, and will use the following settings:
```

Reply:
Was meant to be a mathematical formula, will witch to $$ (think this was written before you told me to use that instead of **)